tests/testthat/test-make-array.R

```diff
@@ -303,4 +345,19 @@ with_test_authentication({
             expect_equivalent(dim(as.vector(ds$mr_5)), c(nrow(ds), 5))
         })
     })
+    
+    whereas("formMR functions as expected", {
+        ds <- newDataset(mrdf)
+        ds$mrVar <- formMR(
+            SubvariableDefinition(ds$v4 == "B", name = "subvar name", alias = "sv_alias"), 
+            name = "MR Variable"
+        )
+        
+        test_that("can make MR with formMR", {
+            expect_identical(names(ds), c(names(mrdf), "mrVar"))
+            expect_true(is.MR(ds$mrVar))
+            expect_true(is.Categorical(ds$mrVar$sv_alias))
+            expect_equal(unclass(as.vector(table(ds$mrVar$sv_alias))), c(2, 2))
+        })
+    })
 })
```

Comment:
![image](https://user-images.githubusercontent.com/1051049/75071711-effe5580-54c3-11ea-89b8-c51567bcebaf.png)
wat

Reply:
(probably try to fix this ordering now; eventually probably see if we can assert it in an integration test)

Reply:
fixed, needed to specify names in value object within map